website/docs/reference/events.mdx

```diff
@@ -0,0 +1,1077 @@
+---
+title: Events
+---
+
+## Overview
+
+Events help you keep track of changes across your Unleash instance. You can explore events in [Event Log](#event-log) or using the global [Event Timeline](#event-timeline).
+
+The Event Log provides a full list of all events, whereas the Event Timeline offers a highlight of recent key events and [Signals](/reference/signals) from external sources.
+You can use Event Log for auditing purposes, while Event Timeline is particularly helpful for debugging recent changes and their impact on your Unleash instance.
```

Comment:
Just to clarify: The impact is not usually felt on your Unleash instance itself. Instead, it's experienced in external systems controlled by your Unleash feature flags.

For example, imagine we start seeing signals on the timeline, like a high error rate from `example-app`. Looking back, we notice these signals only began after enabling `example-flag`. By disabling `example-flag` and confirming that the signals stop, we can infer (or at least suspect) that the feature flag triggered the errors, allowing us to mitigate the issue.